app/controllers/api/v1/motorcycles_controller.rb

```diff
@@ -35,30 +35,33 @@ def create
 
   # UPDATE MOTORCYCLE
   def update
-    # do your work
+    @motorcycle = Motorcycle.find(params[:id])
+    update_cycle = params.require(:motorcycle)
+      .permit(:cylinder, :description, :model, :acceleration, :title, :price, :duration, :discount, images: [])
+      .merge(user: @user)
+    if @motorcycle.update(update_cycle)
```

Comment:
- [ ]  Please first compare the user in the token with the creator of the motorcycle if they are equal update the motorcycle. If they are not equal. Please send an error message. Remember Only the creator of the motorcycle can update the item.